src/docker-in-docker/install.sh

```diff
@@ -468,6 +469,35 @@ if [ "${INSTALL_DOCKER_BUILDX}" = "true" ]; then
     find "${docker_home}" -type d -print0 | xargs -n 1 -0 chmod g+s
 fi
 
+if [ -z "$DISABLE_IP6_TABLES" ]; then
+    DOCKER_DEFAULT_IP6_TABLES=""
+else
+    version_to_check=""
+    semver_regex="^(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)\.(0|[1-9][0-9]*)(?:-([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?$"
+    if echo "$DOCKER_VERSION" | grep -Eq "$semver_regex"; then
+        version_to_check=$(echo $DOCKER_VERSION | cut -d. -f1)
+    elif echo "$DOCKER_VERSION" | grep -Eq "^-?[0-9]+$"; then
+        version_to_check=$DOCKER_VERSION
+    fi
+
+    if [ -n "$version_to_check" ] && [ "$version_to_check" -ge 27 ] || [ "$DOCKER_VERSION" = "latest" ]; then
+            if [ "$DISABLE_IP6_TABLES" == false ]; then
+                DOCKER_DEFAULT_IP6_TABLES=""
```

Comment:
We don't need to handle this separately for > v27 and lower. Let's just set `DOCKER_DEFAULT_IP6_TABLES` to empty by default and only handle `true` cases

Reply:
for the two cases such that v26 and lower, script exits out seeing the lesser version and disableIp6tables value as `false` whereas for v27 and up, script simply does add the flag --ip6tables=false when disableIp6tables equalling `true` is provided by user as the config option, the if conditional statement shall be used